samples/snippets/src/main/java/pubsublite/spark/SimpleRead.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pubsublite.spark;
+
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.streaming.OutputMode;
+import org.apache.spark.sql.streaming.StreamingQuery;
+import org.apache.spark.sql.streaming.Trigger;
+
+public class SimpleRead {
+
+  public static void main(String[] args) throws Exception {
```

Comment:
Please update these to follow the [Method Structure in the Samples Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#method-structure).

Reply:
Can you elaborate a bit? Do you mean I need to extract the majority of main into a separate void method and then call it in "main"?

Reply:
Yep, you got it @jiangmichaellll

Reply:
Right now you have no tests. In order to test it, you'll need to be able to call the function with some configuration so you can test it. The standard way of doing that is to:
1. Move the "meat" of the sample into a single function, which can be called for testing
2. Use the `main` as an example of the configurable bits, which gives users an easy single entrypoint to edit and run the sample


So this sample would probably look something like this:

```java
public static void main(main(String[] args) {
    // TODO(developer): Replace these variables before running the sample.
    String sourceSubscriptionPath = "this-is-an-example-subscription-path" 
    simpleRead(projectId, filePath);
}

// This sample performs a Simple Read using PubSubLite.
public static void simpleRead(String sourceSubscriptionPath) {
    // ...
}
```

Then you should add some tests to verify the sample works. [Here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/storage/s3-sdk/src/test/java/ListGcsBucketsTest.java) is a decent example.

Reply:
I have to use command line argument for SimpleRead, the reason is commented in the SimpleRead.java. The other samples all take environment vars now.